tools/taskcluster/build_targets.py

```diff
@@ -0,0 +1,64 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script builds the release apks using the options passed in.
+"""
+import getopt;
+import subprocess
+import sys
+
+platforms = {
+   'oculusvr': ['arm', 'arm64'],
+   'wavevr': ['arm'],
+   'googlevr': ['arm', 'arm64'],
+   'noapi': ['arm', 'arm64', 'x86'],
+   'svr': ['arm', 'arm64'],
+}
+
+def findMode(arg):
+   values = arg.split(':')
+   if len(values) > 1:
+     mode = values[1].lower()
+     if mode == 'r' or mode == 'release':
+        return 'Release'
+     elif mode == 'd' or mode == 'debug':
+        return 'Debug'
+     elif mode == 'a' or mode == 'all':
+        return '';
+   return 'Release'
+
+def findArch(mode):
+   archList = platforms.get(mode[0], ["arm"])
+   if len(mode) > 1:
+      value = mode[1]
+      if value == 'a' or value == 'all':
+         return archList;
+      elif value.lower() in archList:
+         return [value.lower()]
+   return [archList[0]]
+
+def main(name, argv):
+   targets =['tag']
+   if len(argv) > 0:
+      targets = argv[0].split('-')
+   mode = findMode(targets[0])
+   size = len(targets)
+   if size == 1:
+      for value in platforms.keys():
+         targets.append(value + ':all')
+   command = []
+   for item in targets[1:]:
+      itemList = item.split(':')
+      if itemList[0] not in platforms:
+         print >> sys.stderr, "Error: " + itemList[0] + " is not a supported platform"
```

Comment:
just noticed that this will leak the error messages to the [shell call](https://github.com/MozillaReality/FirefoxReality/pull/1132/files#diff-ac0229d1171c0983b27003af4c7441a5R126)

Reply:
It is sent to stderr so it will show in the taskcluster log with out messing up the print result at the end.